### ISSUE TITLE
Spectrum Analyzer mode

### ISSUE THREAD

**Issue by jkassot** [xaxaxa-dev/vna#28]
Would it be possible to add a Spectrum Analyzer function/mode?
Just scan the desired band and display the input of Rx port.
Maybe could include a "Thru" measurement to calibrate the receiver for power measurement for the Band that will be scanned.

**Comment by satfan52** [comment]
Yes, it would be really great if we could also use the XaVNA as a sweep based spectrum analyser.
Though the current sweeping speed would have to be increased as for spectrum analyser it must be much faster than for a VNA (for obvious reasons). We would also need to be able to contrôle the Raw Bandpass filter and go down to resolutions of 1khz if not 100hz for narrow bandwidth signal analysis.

**Comment by garangstore** [comment]
A [spectrum analyzer](https://www.garangstore.com/spectrum-analyzer) measures the magnitude of an input signal versus frequency within the full frequency range of the instrument. The primary use is to measure the power of the spectrum of known and unknown signals. The input signal that most common spectrum analyzers measure is electrical; however, spectral compositions of other signals, such as acoustic pressure waves and optical light waves, can be considered through the use of an appropriate transducer. Spectrum analyzers for other types of signals also exist, such as optical spectrum analyzers which use direct optical techniques such as a monochromator to make measurements.
